### PR TITLE
DS-3219 -> Word breaking on long filenames/description (review/upload)

### PR DESCRIPTION
**Pull request by jonas-atmire**
JIRA Ticket : https://jira.duraspace.org/browse/DS-3219
XMLUI counterpart for https://github.com/DSpace/DSpace/pull/1420
Long file name or description breaks file upload / review page (Mirage1+2)

### CODE REVIEW

**Comment by hardyoyo** [comment]
+1 by review

**Comment by terrywbrady** [comment]
+1, tested locally and confirmed fix in XMLUI

**Comment by tdonohue** [comment]
Just noticed that the corresponding JSPUI ticket #1420 has had updates by @PhilipVis to the `break-all` styles to make them compatible to all browsers.

@jonas-atmire , while this sounds like it's mostly ready to go, I'm going to wait to merge until we get the same CSS level fixes in this PR.

Thanks!

**Comment by jonas-atmire** [comment]
@tdonohue 
I was going to do the port but hadn't found the time yet.
This updates both Mirage1+2  to be concise with the JSPUI css fix

File: dspace-xmlui-mirage2/src/main/webapp/styles/shared/_dspace-bootstrap-tweaks.scss [hunk 1]
```diff
@@ -139,4 +139,9 @@ span.bold {
 /* Preserve whitespace. Used to display error stack in a readable way. */
 p.pre {
     white-space:pre;
+}
+.break-all{
+  -ms-word-break:   break-all;
+  word-break:       break-word;
```
Comment (pnbecker) [inline dspace-xmlui-mirage2/src/main/webapp/styles/shared/_dspace-bootstrap-tweaks.scss]: While testing #1420 I found that word-break must be set to break-all. (http://www.w3schools.com/cssref/css3_pr_word-break.asp)

**Comment by terrywbrady** [comment]
Confirmed that the latest commit resolved the item submit/review issue in Firefox.

Note that there is an issue when editing an item (exists in Firefox and Chrome)

image

**Comment by terrywbrady** [comment]
Tested in IE and Safari.  The first issue (item submit review) is resolved across browsers.  The edit item issue is present across browsers.

**Comment by jonas-atmire** [comment]
Added the css class to both the overview (EditItemBitstreamForm) and specific edit of the bitstream (EditBitstreamForm) as this was broken as well.

Haven't had time to check for even more places where this could be broken, but it still would require only some minor changes if more pop up

**Comment by terrywbrady** [comment]
Confirmed fix for DS-3223 in the latest commit

**Pull request merged**